lib/cloudware/commands/concerns/model_list.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
```

Comment:
Originally there was going to be `Domain`, `Group`, and `Machine` models that all had the `Tags` behaviour. For a period of time the `Domain` model did exist and the `Tags` concern was being shared.

However `Models::Domain` has since been removed but the `Tags` concern has lingered on. I'll create an issue to refactor this.